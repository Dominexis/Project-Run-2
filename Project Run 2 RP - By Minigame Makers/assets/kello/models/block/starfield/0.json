{
  "parent": "kello:block/cube_particle",
  "textures": {
    "particle": "kello:item/visual/starfield/stars_0",
    "all": "minecraft:block/note_block"
  }
}