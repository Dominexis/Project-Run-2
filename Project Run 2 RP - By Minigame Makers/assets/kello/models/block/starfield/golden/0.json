{
  "parent": "kello:block/cube_particle",
  "textures": {
    "particle": "kello:item/visual/starfield/golden_stars_0",
    "all": "minecraft:block/note_block"
  }
}